### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by biohazd**
adding additional wording so its easier to see there are 3 options to use for Prometheus and Kubernetes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,16 +29,17 @@ post](https://coreos.com/blog/the-prometheus-operator.html).
...
+Prometheus Operator - The Prometheus Operator uses Kubernetes [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to simplifiy the deployment and configuration of Prometheus, Alertmanager, and related monitoring components.
 
-[kube-prometheus](https://github.com/prometheus-operator/kube-prometheus) provides example configurations for a complete cluster monitoring
+kube-prometheus - [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus) provides example configurations for a complete cluster monitoring
 stack based on Prometheus and the Prometheus Operator.  This includes deployment of multiple Prometheus and Alertmanager instances,
 metrics exporters such as the node_exporter for gathering node metrics, scrape target configuration linking Prometheus to various
 metrics endpoints, and example alerting rules for notification of potential issues in the cluster.
 
-The [stable/prometheus-operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator)
+prometheus-operator - The [stable/prometheus-operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator)
```
Comment (lilic) [inline README.md:39]: This was moved to https://github.com/prometheus-community/helm-charts maybe we can point to that instead? Thanks!

File: README.md [hunk 2]
```diff
@@ -29,16 +29,17 @@ post](https://coreos.com/blog/the-prometheus-operator.html).
 
 ## Prometheus Operator vs. kube-prometheus vs. community helm chart
 
-The Prometheus Operator uses Kubernetes [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to simplifiy the deployment and configuration of Prometheus, Alertmanager, and related monitoring components.
+Prometheus Operator - The Prometheus Operator uses Kubernetes [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to simplifiy the deployment and configuration of Prometheus, Alertmanager, and related monitoring components.
 
-[kube-prometheus](https://github.com/prometheus-operator/kube-prometheus) provides example configurations for a complete cluster monitoring
+kube-prometheus - [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus) provides example configurations for a complete cluster monitoring
```
Comment (lilic) [inline README.md:34]: Not sure we need to reference the name twice, the way we had it before is okay.

File: README.md [hunk 3]
```diff
@@ -29,16 +29,17 @@ post](https://coreos.com/blog/the-prometheus-operator.html).
 
 ## Prometheus Operator vs. kube-prometheus vs. community helm chart
 
-The Prometheus Operator uses Kubernetes [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to simplifiy the deployment and configuration of Prometheus, Alertmanager, and related monitoring components.
+Prometheus Operator - The Prometheus Operator uses Kubernetes [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to simplifiy the deployment and configuration of Prometheus, Alertmanager, and related monitoring components.
```
Comment (lilic) [inline README.md:32]: Same here, we can maybe instead add a small header to each section so it reads better, wdyt?

**Review by lilic — COMMENTED** [review]
Thanks for your PR! A couple of comments. :)

File: README.md (hunk 1 above)
Comment (biohazd) [reply README.md:39]: yes that woud be good,

File: README.md (hunk 3 above)
Comment (biohazd) [reply README.md:32]: yes, small headers would be good.    It makes it abit easier to see that there are 3 options.     (I dont know how to do small headings)

Comment (biohazd) [reply README.md]: see if its better now.   is there a way to repview my changes ?

File: README.md [hunk 4]
```diff
@@ -29,16 +29,19 @@ post](https://coreos.com/blog/the-prometheus-operator.html).
...
 The Prometheus Operator uses Kubernetes [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to simplifiy the deployment and configuration of Prometheus, Alertmanager, and related monitoring components.
 
+### kube-prometheus
 [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus) provides example configurations for a complete cluster monitoring
 stack based on Prometheus and the Prometheus Operator.  This includes deployment of multiple Prometheus and Alertmanager instances,
 metrics exporters such as the node_exporter for gathering node metrics, scrape target configuration linking Prometheus to various
 metrics endpoints, and example alerting rules for notification of potential issues in the cluster.
 
-The [stable/prometheus-operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator)
+### prometheus-operator
```
Comment (lilic) [inline README.md:41]: Lets just do helm chart instead, wdyt?
```suggestion
### helm chart
```

**Comment by lilic** [comment]
Do you mind just rebasing and squashing your commits after the change is addressed, otherwise lgtm :) 🎉 Thanks!

**Comment by biohazd** [comment]
> Do you mind just rebasing and squashing your commits after the change is addressed, otherwise lgtm :) 🎉 Thanks!

how do i rebase and squash ?

**Comment by biohazd** [comment]
Does it look ok now?  I think I did  therebase right.

File: README.md [hunk 5]
```diff
@@ -29,16 +29,25 @@ post](https://coreos.com/blog/the-prometheus-operator.html).
...
+### Prometheus Operator
 The Prometheus Operator uses Kubernetes [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to simplifiy the deployment and configuration of Prometheus, Alertmanager, and related monitoring components.
 
+### kube-prometheus
 [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus) provides example configurations for a complete cluster monitoring
 stack based on Prometheus and the Prometheus Operator.  This includes deployment of multiple Prometheus and Alertmanager instances,
 metrics exporters such as the node_exporter for gathering node metrics, scrape target configuration linking Prometheus to various
 metrics endpoints, and example alerting rules for notification of potential issues in the cluster.
 
+<<<<<<< HEAD
```
Comment (lilic) [inline README.md:41]: You should remove this as it seems its leftover from rebase.

File: README.md [hunk 6]
```diff
@@ -29,16 +29,25 @@ post](https://coreos.com/blog/the-prometheus-operator.html).
...
 The [prometheus-community/kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)
 helm chart provides a similar feature set to kube-prometheus. This chart is maintained by the Prometheus community.
 For more information, please see the [chart's readme](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack#kube-prometheus-stack)
+=======
```
Comment (lilic) [inline README.md:45]: This one as well.

File: README.md [hunk 7]
```diff
@@ -29,16 +29,25 @@ post](https://coreos.com/blog/the-prometheus-operator.html).
...
+### helm chart
+The [stable/prometheus-operator](https://github.com/prometheus-community/helm-charts)
+helm chart provides a similar feature set to kube-prometheus. This chart is maintained by the Helm community.
+For more information, please see the [chart's readme](https://github.com/prometheus-community/helm-charts/blob/main/README.md)
+>>>>>>> Update README.md
```
Comment (lilic) [inline README.md:50]: And this one. 🎉

**Review by lilic — COMMENTED** [review]
Almost there, thanks! :)

**Comment by biohazd** [comment]
hows it look now ?

**Review by lilic — APPROVED** [review]
Nice, thanks!

**Pull request merged**